urbansearch/server/main.py

```diff
@@ -0,0 +1,34 @@
+from flask import Flask
+from urbansearch.server.predict import predict_api
+from urbansearch.server.dataset import dataset_api
+
+API_PREFIX = '/api/v1'
+
+
+class Server(object):
+    """
+    Server class which creates a new server and runs it if the run parameter
+    is set to True (default).
+    """
+
+    def __init__(self, run=True):
+        """
+        Constructor
+
+        :param run: Boolean value which tells if the app should be started
+        or not. In case run is set to True or ommited the app is run, else
+        only an app instance is created.
+        """
+
+        self.app = Flask(__name__)
+        self.register_blueprints()
+
+        if run:
+            self.app.run()
+
+    def register_blueprints(self):
+        """
+        Register the predefined blueprints
+        """
+        self.app.register_blueprint(predict_api, url_prefix=API_PREFIX+'/classify')
```

Comment:
I don't think this agrees to PEP8, there should be spaces between the '+'. Seems like a "screw these spaces, I want 80 character line" thingy? :grimacing: